File: JSBASIC/194_union.js
// 194	194_union.js	Find the union of two arrays (all unique elements from both).

let arr1 = [1, 2, 3, 4];
let arr2 = [3, 4, 5, 6];
let union = [...new Set([...arr1, ...arr2])];
console.log("Array 1:", arr1);
console.log("Array 2:", arr2);
console.log("Union:", union);

File: JavaBasic/194_union.java
// 194	194_union.java	Find the union of two arrays (all unique elements from both).
import java.util.*;
public class _194_union {
    public static void main(String[] args) {
        Set<Integer> union=new LinkedHashSet<>(Arrays.asList(1,2,3,4));
        union.addAll(Arrays.asList(3,4,5,6));
        System.out.println("Union: " + union);
    }
}
